JDBC/GetBooks.java

```diff
@@ -0,0 +1,75 @@
+package com.metacube.jdbc.example;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Iterator;
+
+public class GetBooks {
+	public static void main(String[] args) throws IOException {
+		//buffer object to get name of book and member
+		BufferedReader bufferedReaderObj;
+		bufferedReaderObj = new BufferedReader(new InputStreamReader(System.in));
+		System.out.println("Enter Author Name");
+		String authorName = bufferedReaderObj.readLine(); 
+		ArrayList<Titles> titleList = getBooksByAuthorName(authorName);
+		Iterator<Titles> iterator = titleList.iterator();
+		//to check titleList is empty or not
+		if(titleList.size()==0) {
+			System.out.println("No titles corresponding to this author");
+			System.exit(0);
+		}
+		System.out.println("Title Name");
+		while(iterator.hasNext()) {
+			System.out.println(iterator.next().getTitleName()+"\t");
+		}
+		System.out.println("\nExiting . . .");
+	}
+	/* execute query using prepared statement*/
+	//method which return list of titles
+	private static ArrayList<Titles> getBooksByAuthorName(String authorName) {
+		//Fetching data using PreparedStatement
+		Connection con = null;
+		ArrayList<Titles> titleList = new ArrayList<Titles>();
+		PreparedStatement ps = null;
```

Comment:
The basic thing which needs to be understood as per the assignment guidelines is that the API should be so robust that when some one wants any records from the DB, he should simply invoke the API by passing the query and retrieving the result. The intermediate process should be done in the util class.